src/vs/editor/contrib/inlineCompletions/inlineCompletionsModel.ts

```diff
@@ -428,23 +428,35 @@ export interface NormalizedInlineCompletion extends InlineCompletion {
 	range: Range;
 }
 
+function leftTrim(str: string): string {
+	return str.replace(/^\s+/, '');
+}
+
 export function inlineCompletionToGhostText(inlineCompletion: NormalizedInlineCompletion, textModel: ITextModel): GhostText | undefined {
```

Comment:
@hediet This method looks like a great candidate for some unit tests :).

Reply:
You can read minds 😉